### ISSUE TITLE
openmotif formula

### ISSUE THREAD

**Issue by oliviersoares** [Homebrew/homebrew-x11#248]
I'm getting this error when installing openmotif:

```
==> ./autogen.sh --prefix=/usr/local/Cellar/openmotif/2.3.4 --disable-dependency-tracking --disable-silent-rules --disable-jpeg --disable-png
==> make install
Error: No such file or directory - (/usr/local/Cellar/openmotif/2.3.4/share/man/man3/CORE.3, /usr/local/Cellar/openmotif/2.3.4/share/man/man3/openmotif-CORE.3)
```

Seems like the file:
https://github.com/Homebrew/homebrew-x11/blob/master/openmotif.rb#L62
is missing (/usr/local/Cellar/openmotif dir is missing).

I'm on macOS 10.12. Was able to install on 10.11.
Thanks.

**Comment by apjanke** [comment]
This may have been fixed by #250. Could you `brew update` and try installing again?

**Comment by oliviersoares** [comment]
@apjanke yes it seems fixed now thanks.

oliv@oliv:~$ ll /usr/local/Cellar/openmotif/2.3.6/lib/libXm*
-r--r--r--  1 oliv  staff   2.2M Oct  5 07:47 /usr/local/Cellar/openmotif/2.3.6/lib/libXm.4.dylib
-r--r--r--  1 oliv  staff   3.3M Jun 10 19:06 /usr/local/Cellar/openmotif/2.3.6/lib/libXm.a
lrwxr-xr-x  1 oliv  staff    13B Jun 10 19:06 /usr/local/Cellar/openmotif/2.3.6/lib/libXm.dylib@ -> libXm.4.dylib

However, when i try to install nedit (which is the one I was installing in the first place, openmotif coming as a dependency), it seems to not find the library when trying to compile it.

Here's the full log:

==> Installing nedit from homebrew/x11
==> Downloading https://downloads.sourceforge.net/project/nedit/nedit-source/nedit-5.6a-src.tar.gz
Already downloaded: /Users/oliv/Library/Caches/Homebrew/nedit-5.6a.tar.gz
==> Patching
patching file util/motif.c
==> make macosx MOTIFLINK='-lXm'
Last 15 lines from /Users/oliv/Library/Logs/Homebrew/nedit/01.make:
OpenMotif 2.3
OpenMotif is available from:
	http://www.opengroup.org/openmotif/
	http://www.ist.co.uk/DOWNLOADS/motif_download.html

Also, unless you need a customized NEdit you should STRONGLY
consider downloading a pre-built binary from http://www.nedit.org,
since these are the most stable versions.

If you really want to build an untested version of NEdit you
can override this sanity check by adding -DBUILD_UNTESTED_NEDIT
to the CFLAGS variable in your platform's Makefile (e.g.
makefiles/Makefile.linux)
make[1]: *** [check_tif_rule] Error 1
make: *** [macosx] Error 2

READ THIS: https://git.io/brew-troubleshooting
If reporting this issue please do so at (not Homebrew/brew):
  https://github.com/Homebrew/homebrew-x11/issues

**Comment by apjanke** [comment]
Could you post the full build logs as a gist by running `brew gist-logs nedit`?

**Comment by oliviersoares** [comment]
@apjanke https://gist.github.com/2e825eff5ca18dca861f88bf36f86657

**Comment by apjanke** [comment]
I can reproduce. Looks like `nedit` is not happy building against OpenMotif 2.3.6, which we recently upgraded to.

From the build logs:

```
./check_lin_tif
ERROR:  Untested Motif Version:
	@(#)Motif Version 2.3.6
You are attempting to build NEdit with a version of Motif that
has not been verified to work well with NEdit.  This could be fine,
but it could also lead to crashes and instability.  Historically, 
older versions of Motif have quite often been more stable
than newer versions when used with NEdit, so don't assume newer
is better.

NEdit is known to work with Motif versions:
0.92.32
0.93.0
0.93.12
0.93.18
OpenMotif 2.1.30
OpenMotif 2.2.3
OpenMotif 2.3

...
If you really want to build an untested version of NEdit you
can override this sanity check by adding -DBUILD_UNTESTED_NEDIT
to the CFLAGS variable in your platform's Makefile (e.g.
makefiles/Makefile.linux)
```

We might need to roll back to OpenMotif 2.3.4, or update our patch to `nedit` to let it work with 2.3.6.

I'll take a look at nedit's documentation to see which is the right choice. I've posted on the upstream bug report asking about 2.3.6.

**Comment by apjanke** [comment]
I've merged #256, which hacks nedit to allow openmotif 2.3.6. This nedit issue should be resolved now.
